layout/css/main.css

```diff
@@ -334,9 +369,6 @@ footer{
 		width:960px;
 	}
 	
-.svgFig {
```

Comment:
@mwernimont why don't the svgFigs get resized anymore? Or am I not understanding this?